test/message/core_line_numbers.out

```diff
@@ -12,4 +12,4 @@ RangeError: Invalid input
     at tryModuleLoad (internal/modules/cjs/loader.js:*:*)
     at Function.Module._load (internal/modules/cjs/loader.js:*:*)
     at Function.Module.runMain (internal/modules/cjs/loader.js:*:*)
-    at startup (internal/bootstrap/node.js:*:*)
```

Comment:
These stack frames are truncated because we have a default `Error.stackTraceLimit = 10`. Makes me wonder again that we probably should implement https://github.com/nodejs/node/issues/20505